package.json

```diff
@@ -27,6 +27,8 @@
     "flexsearch": "0.6.32",
     "flux": "4.0.1",
     "font-awesome": "4.7.0",
+    "form-data": "3.0.0",
```

Comment:
These are all dependencies of mattermost-redux that we need to include ourselves now. Once we get the actual monorepo infrastructure sorted out, these will move back into the subpackages.